### DISCUSSION TITLE
[Feature] <app add http basic auth for account password input and request carry function>

### DISCUSSION THREAD

**Discussion by YosenFree** [immich-app/immich#5816] [Feature Request]
### The feature

My immich service is built on a machine without a public network ip and then uses intranet penetration to provide the service. For security reasons, the intranet services have added a http basic authentication, which is required to use the service, otherwise the access will be 401. would it be possible to consider adding a basic auth account password global setting, just set it once, save it on the client, and carry it in the header afterwards.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #3204
